src/scripts.js

```diff
@@ -87,41 +122,42 @@ function getBookingData(customer) {
     const dateMonth = date.getUTCMonth() + 1
     const dateYear = date.getUTCFullYear()
     const currentDate = dateYear + "-" + dateMonth + "-" + dateDay
-    let pic
     sidebar.innerHTML = `<h4>Total Spent: $${customer.totalSpent}</h4>`
+    let pic
     customer.bookings.forEach( booking => {
-    if (!booking.pic) {
-    pic = roomImages[Math.floor(Math.random() * roomImages.length)]
-    booking.pic = pic.toString()
-    }
- 
+    pic = roomImages.find(image => image.Type === booking.roomBooked.roomType)
+   booking.picture
+   booking.picture = pic.picture
+   
+    console.log("pic ", booking.picture)
     let bookingDate = booking.date.split('/').join('-')
```

Comment:
I was forgetting to get the image from booking.roomBooked.roomType, was trying booking.roomType which was the source of my bug